web3/geth.py

```diff
@@ -0,0 +1,20 @@
+from web3.module import (
+    Module,
+)
+from web3.personal import (
+    Personal,
+)
+
+
+class GethPersonal(Personal):
+    """
+    https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal
+    """
+    def __init__(self, w3):
+        self.w3 = w3
+
+
+class Geth(Module):
+    @property
+    def personal(self):
+        return GethPersonal(self.web3)
```

Comment:
Im not sure if it is better, but submodules could be configured in Web3 along with the modules.

Something to the effect of this in web3.Web3:

```python
def get_default_modules():
    return [
        {"name": "eth", "module": Eth},
        {"name": "net", "module": Net},
        {"name": "geth", "module": Geth, "submodules": {"personal": Personal}}
        {"name": "personal", "module": Personal},
        {"name": "version", "module": Version},
        {"name": "txpool", "module": TxPool},
        {"name": "miner", "module": Miner},
        {"name": "admin", "module": Admin},
        {"name": "parity", "module": Parity},
        {"name": "testing", "module": Testing},
    ]

class Web3:
   [...]
    def __init__(self, provider=None, middlewares=None, modules=None, ens=empty):
        self.manager = self.RequestManager(self, provider, middlewares)

        if modules is None:
            modules = get_default_modules()

        for module in modules:
            module['module'].attach(self, module.get('name'))
            if 'submodules' in module:
                for subname, submodule in module['submodules'].items():
                    submodule.attach(getattr(self, module['name']), submodule)

        self.ens = ens
```

The Module.attach method appears to work with modules 2 levels deep. Im leaning toward having 1 way modules are attached.